azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/implementation/PageImpl1.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ */
+
+package com.microsoft.azure.management.storage.implementation;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.microsoft.azure.Page;
+import java.util.List;
+
+/**
+ * An instance of this class defines a page of Azure resources and a link to
+ * get the next page of resources, if any.
+ *
+ * @param <T> type of Azure resource
+ */
+public class PageImpl1<T> implements Page<T> {
+    /**
+     * The link to the next page.
+     */
+    @JsonProperty("nextLink")
```

Comment:
@jejner This is new class named `PageImpl1` for those have next link in the response.